fiftyone/service/ipc.py

```diff
@@ -109,7 +109,7 @@ def send_request(port, message):
     Returns:
         response (any type)
     """
-    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-    s.connect(("localhost", port))
-    pickle.dump(message, s.makefile("wb"))
-    return pickle.load(s.makefile("rb"))
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        s.connect(("localhost", port))
```

Comment:
Indentation issue here?

Reply:
not sure what you mean? It's supposed to be tabbed

Reply:
Ah okay, I misunderstood this github diff to be spaces.